common/rdm/DiscoveryAgent.cpp

```diff
@@ -371,7 +371,7 @@ void DiscoveryAgent::HandleCollision() {
   UID mid_uid(mid >> 32, mid);
   mid++;
   UID mid_plus_one_uid(mid >> 32, mid);
-  OLA_INFO << "Collision, spliting into: " << lower_uid << " - " << mid_uid
+  OLA_INFO << "Collision, splitting into: " << lower_uid << " - " << mid_uid
```

Comment:
This isn't Debian-specific, is it? ;-)

might want to add that separate to these series.

Reply:
Well it was found via https://lintian.debian.org/tags/spelling-error-in-binary.html and confusingly missed via spellintian (I guess because it's in a newer dictionary than I'm using) and codespell.